### PR TITLE
Refactor/cleanup development currency

### PR DESCRIPTION
**Pull request by shmidt-i**

### CODE REVIEW

File: src/Atoms/Currency/Currency.tsx [hunk 1]
```diff
@@ -18,48 +17,5 @@ const FC: React.FC<BaseProps & InjectedIntlProps> = ({ value, currency, intl, si
...
-Title3.displayName = 'Currency.Title3';
-
-export const Currency: CurrencyComponent = {
-  Primary,
-  Secondary,
-  Tertiary,
-  Title1,
-  Title3,
-};
+export const Currency: CurrencyComponent = injectIntl(FC);
```
Comment (JacobBlomgren) [inline src/Atoms/Currency/Currency.tsx]: You can rename `FC` now, maybe call it `FormattedCurrency`? (which it stands for).

File: src/Atoms/Development/Development.tsx [hunk 2]
```diff
@@ -21,41 +21,18 @@ const getColor = (value: number) => (t: Theme) => {
   return t.color.text;
 };
 
-const FD: React.FC<DevelopmentProps & InjectedIntlProps> = ({ value, intl }) => (
-  <>
+const StyledDevelopment = styled.span<DevelopmentProps>`
+  color: ${p => getColor(p.value)(p.theme)};
```
Comment (JacobBlomgren) [inline src/Atoms/Development/Development.tsx]: why higher order?

File: src/Atoms/Currency/Currency.tsx (hunk 1 above)
Comment (shmidt-i) [reply src/Atoms/Currency/Currency.tsx]: fixed, also same in development

File: src/Atoms/Development/Development.tsx (hunk 2 above)
Comment (shmidt-i) [reply src/Atoms/Development/Development.tsx]: changed

**Pull request merged**